docs/testing/automated_benchmark_test.md

```diff
@@ -0,0 +1,110 @@
+# When to perform automated benchmarking tests
+Automated benchmarking tests are additional functional and performance tests to the existing TAP device-based ones. This test suit relies on a configured environment including hypervisors and started VMs, as well as configured SSH authorized key of the execution machine starting the benchmarking tests.
+This benchmarking testing suite is built upon and utilises 1) dpservice-cli to configure running dpservice instances or containers on hypervisors; 2) connectivity and performance testing script under the repository `hack/connectiviy_test`. In the end, running these benchmarking tests is useful for verifying if dpservice works correctly together with actual running VMs for both offloading and non-offloading modes. It also verifies if networking performance meets specified values during dpservice development.
+
+# Examplary command to invoke tests
+```
+./runtest.py --mode offload --stage cicd --docker-image ghcr.io/ironcore-dev/dpservice:sha-e9b4272 -v
+
+./runtest.py --mode both --stage dev -v
```

Comment:
Just a thought for maybe easier operations, that we could rename the TAP script to `taptest.py` and this one to `vmtest.py` or something else, just to prevent confusion in the future?